### PR TITLE
Simple WebViewWidget implementation [android, ios]

### PR DESCRIPTION
**Pull request by Tetraquark**
#55: WebViewWidget implementation for Android and iOS.

### CODE REVIEW

**Comment by Tetraquark** [comment]
The request is currently in a work in progress state.

**Comment by Tetraquark** [comment]
The request is ready for final review.

File: widgets/src/iosMain/kotlin/dev/icerock/moko/widgets/factory/WebViewFactory.kt [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+import platform.Foundation.NSURL
+import platform.WebKit.WKNavigation
+import platform.darwin.NSObject
+
+actual class WebViewFactory actual constructor(
+    private val margins: MarginValues?,
+    private val background: Background?
+) : ViewFactory<WebViewWidget<out WidgetSize>> {
+
+    private val webViewNavDelegate = NavigationDelegate()
```
Comment (Alex009) [inline widgets/src/iosMain/kotlin/dev/icerock/moko/widgets/factory/WebViewFactory.kt]: factory is singletone, but each webview widget is not. you can't save in factory context something which used by widget itself.
to store strong pointer to delegate you can use `setAssociatedObject(object = webView, value = delegate)`

Comment (Tetraquark) [reply widgets/src/iosMain/kotlin/dev/icerock/moko/widgets/factory/WebViewFactory.kt]: Fixed with `setAssociatedObject`.

**Pull request merged**